### PR TITLE
/x/at/calendar/list/search should be quiet if no events are found

### LINKED ISSUE #1059: at/list/search should return nothing if no matching events are found
at/calendar/list currently returns "No events\n" if no events are found.  search returns "\n".  I suggest at, list and search are consistent and return "" if they have nothing to say.  I don't have a strong opinion on calendar.

### PR DESCRIPTION
**Pull request by allanwind**
Fix #1059

### CODE REVIEW

**Comment by geier** [comment]
lgtm. @WhyNotHugo  what do you think?

**Comment by geier** [comment]
Sleeping about this, we probably should at least add some debug logging, that no events were found.

File: tests/cli_test.py [hunk 1]
```diff
@@ -104,7 +104,7 @@ def test_direct_modification(runner):
     runner = runner()
 
     result = runner.invoke(main_khal, ['list'])
-    assert result.output == 'No events\n'
+    assert result.output == ''
```
Comment (WhyNotHugo) [inline tests/cli_test.py:107]: This change should make scripted usage a lot easier. :+1:

File: tests/cli_test.py [hunk 2]
```diff
@@ -245,7 +245,7 @@ def test_calendar(runner):
         assert not result.exception
         assert result.exit_code == 0
         output = '\n'.join([
-            "    Mo Tu We Th Fr Sa Su     No events",
```
Comment (WhyNotHugo) [inline tests/cli_test.py:248]: Not sure I'm convinced about rendering removing this one -- we're rendering the calendar anyway, so it looks kinda odd that there's nothing on the side.

I can't imagine this output being parsed by scripts either; it's far from ideal for being parsed.

**Pull request merged**